Crashlytics/Shared/FIRCLSByteUtility.h

```diff
@@ -15,11 +15,11 @@
 #import <Foundation/Foundation.h>
 
 /**
- * Returns a SHA1 Hash of the input NSData
+ * Returns a SHA1 Hash of the input `Data`
```

Comment:
Not totally necessary to change to Swift data types for internal headers, but probably fine.